test/ui-test/testSuites/suite_status/shared/scripts/names.py

```diff
@@ -2,168 +2,60 @@
 
```

Comment:
As the title is clean names.py. Could you try to split this file and use import?

Maybe 1 file per section + 1 global?

Reply:
I didn't know we could do that! On it

Reply:
Done